system/Session/Handlers/MemcachedHandler.php

```diff
@@ -45,9 +46,18 @@ class MemcachedHandler extends BaseHandler
      * Number of seconds until the session ends.
      *
      * @var int
+     * 
+     * @deprecated
      */
     protected $sessionExpiration = 7200;
 
+    /**
+     * Number of seconds until the session ends.
+     *
+     * @var int
+     */
+    protected $lifetime = 7200;
```

Comment:
I think instead of replicating across all other handlers you can define this instead in BaseHandler. Even if there are handlers not using this, at least it is there available.